src/code/z_actor.c

```diff
@@ -583,7 +583,7 @@ void Target_Draw(TargetContext* targetCtx, PlayState* play) {
 
                     gDPSetPrimColor(OVERLAY_DISP++, 0, 0, entry->color.r, entry->color.g, entry->color.b, (u8)alpha);
 
-                    Matrix_RotateZS(targetCtx->rotZTick * 0x200, MTXMODE_APPLY);
+                    Matrix_RotateZS(targetCtx->reticleSpinCounter * 0x200, MTXMODE_APPLY);
 
                     // Draw the 4 lock-on triangles
```

Comment:
Doesn't show up here, but I think it would make sense to rename `gZTargetLockOnTriangleDL` -> `gLockOnReticleTriangleDL` like OoT in this PR.